src/engine/nodes/OSCSenderNode.cpp

```diff
@@ -107,14 +167,12 @@ void OSCSenderNode::render (AudioSampleBuffer& audio, MidiPipe& midi)
 
     while (iter1.getNextEvent (msg, frame))
     {
-        OSCMessage oscMsg = Util::processMidiToOscMessage (msg);
-        oscSender.send (oscMsg);
-
-        if (oscMessages.size() > maxOscMessages)
-            oscMessages.erase ( oscMessages.begin() );
-        oscMessages.push_back ( oscMsg );
+        msg.setTimeStamp ( timestamp + frame );
```

Comment:
frame should probably be converted to milliseconds
` ((double)frame / sampleRate) * 1000.0)` or something like that

Reply:
Alrighty..